components/access-management/kfam/bindings.go

```diff
@@ -55,19 +57,20 @@ type BindingClient struct {
 
 //getBindingName returns bindingName, which is combination of user kind, username, RoleRef kind, RoleRef name.
 func getBindingName(binding *Binding) (string, error) {
+	// Only keep lower case letters, replace other with -
```

Comment:
I believe we keep lower case letters and numbers with this regex.
Can you update the comment to reflect that?

Reply:
changed